MSSQL/OMOPLoader.sql

```diff
@@ -1805,15 +1805,17 @@ cast(m.start_Date as datetime) measurement_datetime,
 -- NOTE: Are these concept_ids the ones analysts want?
 CASE isnull(m.VALUEFLAG_CD,'0') WHEN 'H' THEN '45876384' WHEN 'L' THEN '45881666' WHEN 'A' THEN '45878745' WHEN 'N' THEN '45884153' ELSE '0' END value_as_concept_id,
 CASE WHEN m.ValType_Cd='N' THEN m.NVAL_NUM ELSE null END value_as_number,
-isnull(m.Units_CD,'') unit_source_value, 
+-- NOTE: Units are pulled from PHS Ontology first (extracted from XML) , if not present then it falls back to units_cd on the observation fact

```

Comment:
Please elaborate to include what it is actually doing (pulling from i_unit in ontology vs units_cd in fact table)

Reply:
Added more elaborated comments.